behind-the-veil-siteroot/imports/ui/components/map/utils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * File Description: API to find coordinates from location input
+ * File version: 1.0
+ * Contributors: Laura
+ */
+
```

Comment:
since this utils file is under the "map" package containing code relevant to the map, it could be name something like mapUtils for now